endpoints/getting-started-grpc/README.md

```diff
@@ -0,0 +1,130 @@
+# Endpoints Getting Started with gRPC & Java Quickstart
+
+It is assumed that you have a working gRPC and Java environment.
+
+1. Build the code:
+
+    ```
+    ./gradlew build
+    ```
+
+1. Test running the code, optional:
+
+    ```
+    # In the background or another terminal run the server:
+    java -jar server/build/libs/server.jar
+
+    # Check the client parameters:
+    java -jar client/build/libs/client.jar --help
+
+    # Run the client
+    java -jar client/build/libs/client.jar --greetee 'Endpoints!'
+    ```
+
+1. Generate the `out.pb` from the proto file.
+
+    ```
+    protoc --include_imports --include_source_info api/src/main/proto/helloworld.proto --descriptor_set_out out.pb
+    ```
+
+1. Edit, `api_config.yaml`. Replace `MY_PROJECT_ID` with your project id.
+
+1. Deploy your service config to Service Management:
+
+    ```
+    gcloud service-management deploy out.pb api_config.yaml
+    # The Config ID should be printed out, looks like: 2017-02-01r0, remember this
+
+    # set your project to make commands easier
+    GCLOUD_PROJECT=<Your Project ID>
+
+    # Print out your Config ID again, in case you missed it
+    gcloud service-management configs list --service hellogrpc.endpoints.${GCLOUD_PROJECT}.cloud.goog
```

Comment:
This is correct. Endpoints needs a globally unique name for the service. We were previously using <project>.appspot.com, but that was causing confusion, as it's not necessarily using AppEnine, and is actually setting up DNS to wherever your API is running. <project>.cloud.goog is a new namespace available, so all the endpoints docs are using <apiname>.endpoints.<project>.cloud.goog